react-boilerplate/actions/index.js

```diff
@@ -0,0 +1,64 @@
+import axios from 'axios';
+import _ from 'underscore';
+import { LOAD_DATA } from './types';
+
+const TERMINAL_URL = 'http://localhost:10080/api/v1/terminal';
+let pollId,
+		lastParams,
+		dispatch;
+
+function action(actionType, payload) {
+
+	return {
+		type: actionType,
+		payload,
+	};
+
+}
+
+export function getTerminalData(params = {}) {
+
+	lastParams = params;
+	const query = _.isEmpty(params) ? params : { params };
+
+	axios.get(TERMINAL_URL, query)
+		.then(response => {
+
+			dispatch(action(LOAD_DATA, response.data));
+
+		})
+		.catch((error) => {
+
+			console.log(error);
+
+		});
+
+}
+
+export function startPollServer() {
+
+	if (typeof pollId == 'undefined') {
+
+		pollId = setInterval(() => getTerminalData(lastParams), 3000);
+
+	}
+
+}
+
+export function stopPollServer() {
+
+	if (typeof pollId != 'undefined') {
+
+		clearInterval(pollId);
+		pollId = undefined;
+
+	}
+
+}
+
+export function init(storeDispatch) {
+
+	dispatch = storeDispatch;
+	startPollServer();
+
+}
```

Comment:
Is it possible to add comments in this file to explain:
- the purpose of the file
- role of each function